### ISSUE TITLE
Add docker run support to ECS

### ISSUE THREAD

**Issue by chris-crone** [docker/compose-cli#829]
**Description**
The Compose CLI currently only supports using Compose to deploy workloads to ECS using `docker compose` commands. There may be use cases where users would like to run simple workloads against ECS using the `docker run` command.

Please leave us feedback on if you think this would be useful and give an example of what your use case would be.

**Comment by mattyweb** [comment]
we have a nightly data ingestion workflow that runs in an ephemeral container. it spins up, add some data to another container, and spins down. we're looking to move to use the docker compose cli with ECS and would love to keep using docker run for this.

**Comment by chloerei** [comment]
Yes, this is useful. I need to use the `docker run` command from development to production environment.

-In the development environment, I need to execute commands like `bin/rails db:migrate` `bin/rails test` in container.
-In a production environment, I need to run commands like `bin/rails console` `bin/rails runner script.rb` to perform temporary tasks or data checks.

I am very excited to see that docker compose supports deployment to ECS. The lack of `run` is the reason why I failed to switch from docker-compose. Hope to implement this function.
